eng/native/init-distro-rid.sh

```diff
@@ -138,7 +138,8 @@ initDistroRidGlobal()
         local distroRid=""
 
         # Check for musl-based distros (e.g Alpine Linux, Void Linux).
-        if "${rootfsDir}/usr/bin/ldd" --version 2>&1 | grep -q musl; then
+        if "${rootfsDir}/usr/bin/ldd" --version 2>&1 | grep -q musl ||
+           file "${rootfsDir}/usr/bin/ldd" 2>&1 | grep -q musl; then
```

Comment:
Could also check the same thing with `strings`:

```sh
if "/crossrootfs/arm64/usr/bin/ldd" --version 2>&1 | grep -q musl ||
    strings "/crossrootfs/arm64/usr/bin/ldd" 2>&1 | grep -q musl ||
    file "/crossrootfs/arm64/usr/bin/ldd" 2>&1 | grep -q musl; then
```

Reply:
I would keep just the way with "strings (without even trying to run ldd). It should cover all the cases

Reply:
@janvorli, unfortunately in voidlinux `strings(1)` and `file(1)` are not available in the base image. So keeping the --version checking would help. `file` is redundant, so I have removed it.

Reply:
Ok, makes sense.